dataclients/kubernetes/clusterstate.go

```diff
@@ -22,8 +22,6 @@ type clusterState struct {
 }
 
 func (state *clusterState) getService(namespace, name string) (*service, error) {
-	state.mu.Lock()
-	defer state.mu.Unlock()
```

Comment:
Why is this removed if we did not land https://github.com/zalando/skipper/pull/2080

Lets wait for it maybe?

Reply:
ok